homeassistant/components/influxdb.py

```diff
@@ -81,6 +81,7 @@
 def setup(hass, config):
     """Set up the InfluxDB component."""
     from influxdb import InfluxDBClient, exceptions
+    import requests.exceptions
```

Comment:
This is a core requirements and would be imported on top

Reply:
Oh, right. Thanks for pointing that out.